src/JWT/Algorithms/RS256Algorithm.cs

```diff
@@ -19,8 +19,8 @@ public sealed class RS256Algorithm : IJwtAlgorithm
         /// <param name="privateKey">The RSA key for signing the data.</param>
         public RS256Algorithm(RSACryptoServiceProvider publicKey, RSA privateKey)
         {
-            _publicKey = publicKey ?? throw new InvalidOperationException("Private key is null");
-            _privateKey = privateKey ?? throw new InvalidOperationException("Public key is null");
+            _publicKey = publicKey ?? throw new ArgumentNullException("Public key is null");

```

Comment:
Pass parameter name to first argument, rather than message. You can pass both too, see https://docs.microsoft.com/en-us/dotnet/api/system.argumentnullexception.-ctor?view=netframework-4.8#System_ArgumentNullException__ctor_System_String_System_String_

Reply:
Thanks! I fixed it.